src/components/pages/career-details/career-form/utils.ts

```diff
@@ -0,0 +1,19 @@
+const getFilenameWithoutFiletype = (filename) => {
```

Comment:
we don't need to expose it, the fileId function is what we are interested in. so it stays local.